docs/administration/vulnerability-scanning/deployment-security.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Deployment security
+weight: 31
+---
+
+From version 2.0, Harbor enables capabilities checking for the pluggable scanners. It will use the **consumes_mime_types** of capabilities from scanner metadata to decide whether the artifact supported by the scanner. That means some kinds of artifacts may never be supported by the scanner, can some kinds of artifacts will be supported by the scanner. For example, the artifacts its type is helm chart is not supported by the scanner currently.
+
+The result is that Harbor will skip the **Deployment security** option in the project **Configuration** for the artifacts which are not supported by the scanner of the project.
```

Comment:
change line 8 and line 10 to the following one paragraph 

'Harbor v2.0 now supports OCI image index, which is a higher-level manifest which points to specific image manifests, ideal for one or more platform.  Scanning for OCI image index is also supported, with the scan result of the index being an aggregation of the scan results of the artifacts referenced within.

Harbor has ‘deployment security’ which can prevent artifacts from being pulled if vulnerabilities are discovered. For pulling indexes, ‘deployment security’ will skip this policy checking for the index artifact itself and will only apply policy checking on the referenced artifacts and at the individual artifact level and not on the index as a whole. This means when pulling Redis for ARM for example, it only checks to see if whether Redis for ARM has vulnerabilities and not impacted by whether amd64 has CVEs. This applies to CNABs as well.  '